pkg/oc/admin/migrate/storage/storage.go

```diff
@@ -198,6 +208,8 @@ func NewCmdMigrateAPIStorage(name, fullName string, f *clientcmd.Factory, in io.
 	flags.IntVar(&options.bandwidth, "bandwidth", 10,
 		"Average network bandwidth measured in megabits per second (Mbps) to use during storage migration.  Zero means no limit.  This flag is alpha and may change in the future.")
 
+	flags.BoolVar(&options.skipVersionCheck, "skip-versioncheck", false, "Skip minor version check between client and server.")
```

Comment:
@deads2k  I think we don't want to do this skip at all. The migration always needs to be performed using the exact version, or same and higher?